types/fluent__dedent/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for @fluent/dedent 0.1
+// Project: http://projectfluent.org
+// Definitions by: Mark Weaver <https://github.com/blushingpenguin>, Huy Nguyen <https://github.com/huy-nguyen>, James Nimlos <https://github.com/jamesnimlos>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+// tslint:disable-next-line:npm-naming
+export default function ftl(strings: TemplateStringsArray, ...values: any[]): string;
```

Comment:
Looks like this shouldn’t be a default export; it’s a UMD wrapping a CommonJS export: https://unpkg.com/browse/@fluent/dedent@0.1.0/index.js

```ts
declare function FluentDedent(/* ... */): string;
export = FluentDedent;
export as namespace FluentDedent;
```

Reply:
Ok, I see what you mean -- I have fixed the dedent definition and added namespaces to the other modules now.  Also I had originally split the definition files as per the module source which was wrong as the packaged modules are all in one file so I've merged them back into one.
Thanks for the help!